static/third/bootstrap-tooltip/tooltip.css

```diff
@@ -5,7 +5,7 @@
     display: block;
     padding: 5px;
     font-size: 11px;
-    opacity: 0;
+    opacity: 1!important;
```

Comment:
@nikhilmaske-2001 I think we should avoid using `!important` if there's a workaround orelse we can write the selectors it ll affect & reason for its necessity in comments.